client/src/components/Footer.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Logo from './Logo';
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <Logo />
+      <div className="footer__caption">UNI SOLUTIONS</div>
+      <div className="footer__copyright">&copy; UNI SOLUTIONS 2022</div>
```

Comment:
rok raczej powinien być dynamicznie wyliczany

UNI Solutions - też się pojawia w wielu miejscach - może warto z tego zrobić const'a - ale to akurat pierd